src/components/infinite-scroll/infinite-scroll.ts

```diff
@@ -193,6 +194,21 @@ export class InfiniteScroll {
   }
 
   /**
+   * @input {string} The position of the infinite scroll element.
+   * The value can be either `top` or `bottom`.
+   * Default is `bottom`.
+   */
+  @Input()
+  get position(): string {
+    return this._position;
+  }
+  set position(val: string) {
+    if (val === POSITION_TOP || val === POSITION_BOTTOM) {
+      this._position = val;
+    }
```

Comment:
let's add here a `console.error()` telling val was an invalid value, and the two possible valid values.